specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -128,16 +129,42 @@ Given these elements, the number of test cases for each scenario can be calculat
 **Alternate flows**
 
 1. The skill creates a conversation (in teams) and starts a 1:1 conversation with a user in the group. Note - the 1:1 conversations created persist, and there is no way to delete them. Repeated calls to createConversation will succeed however, and return the appropriate conversationId that can be re-used.
+2. **NOTE: Are there meaningful differences in other channels between create and continue conversation?**
 
 **Total test cases:** 96 (not including alternate flows)
 
+### 3a. Skill sends a proactive message to consumer - create conversation
+
+> A consumer calls a _reminder skill_ and the skill finishes the conversation but stores some tasks. Then, some time later the skill creates a new conversation with the consumer and sends a message.
+
+
+**Testing matrix**
+
+- Skill: RemindMe
+- Topology: [Simple](#simple)
+
+![Bot SDLC](media/Simple.jpg)
+
+**Variables**
+
+- Auth context: Public Cloud, Gov Cloud, Sandboxed
+- Delivery mode: Normal, ExpectReplies
+
+**Alternate flows**
+
+1. The skill creates a conversation (in teams) and starts a 1:1 conversation with a user in the group. Note - the 1:1 conversations created persist, and there is no way to delete them. Repeated calls to createConversation will succeed however, and return the appropriate conversationId that can be re-used.
+
+
+**Total test cases:** 96 (not including alternate flows)
+
+
 ### 4. Card actions that generate invokes and message activities
 
 > The Consumer invokes the skill through a dialog, and the Skill sends an Adaptive Card that collects some information with a "Submit" button using Action.Submit as the action type.
 
 **Testing matrix**
 
-- Skill: TBD
+- Skill: [Consolidate card functions](https://fuselabs.visualstudio.com/TeamsIntegrationTesting/_git/TeamsIntegrationTesting?path=%2FDotnetIntegrationBot%2FIntegrationCommandHandler.cs)
```

Comment:
I wouldn't include references to our internal repos. I also think this scenario is for card actions in general and not just teams

Reply:
resolved